library/src/scala/quoted/Expr.scala

```diff
@@ -6,10 +6,14 @@ import scala.quoted.show.SyntaxHighlight
 class Expr[+T] private[scala] {
 
   /** Show a source code like representation of this expression without syntax highlight */
-  def show(using qctx: QuoteContext): String = qctx.show(this, SyntaxHighlight.plain)
+  def show(using qctx: QuoteContext): String =
+    import qctx.tasty.{_, given _}
+    this.unseal.showWith(SyntaxHighlight.plain)
 
   /** Show a source code like representation of this expression */
-  def show(syntaxHighlight: SyntaxHighlight)(using qctx: QuoteContext): String = qctx.show(this, syntaxHighlight)
+  def show(syntaxHighlight: SyntaxHighlight)(using qctx: QuoteContext): String =
+    import qctx.tasty.{_, given _}
+    this.unseal.showWith(syntaxHighlight)
```

Comment:
Minor: a complex import for just one line of code does not look nice.

Reply:
Should be able to remove with #8502. I will rebase and check.

Reply:
Indeed. I removed the imports.